cosmo_tester/test_suites/cluster/full_cluster_test.py

```diff
@@ -183,6 +183,70 @@ def _wait_for_cfy_node_to_start_serving(cfy, timeout=15):
     raise TimeoutException
 
 
+def test_workflow_resume_manager_failover(minimal_cluster, cfy):
+    broker, db, mgr1, mgr2 = minimal_cluster
+    mgr1.use(cert_path=mgr1.local_ca)
+    plugin_name = 'waiting'
+    plugin_yaml = '{}-blueprint.yaml'.format(plugin_name)
+    plugin_wagon = 'testplugin-0.0.0-py27-none-any.wgn'
+
+    # upload a mock plugin which waits 1 minute, deploy and execute
+    tests_root = path.dirname(path.dirname(path.dirname(
+        path.realpath(__file__))))
+    blueprints_dir = path.join(tests_root, 'resources', 'blueprints', 'mocks')
+    cfy.blueprints.upload(path.join(blueprints_dir, plugin_yaml),
+                          '-b', plugin_name)
+    cfy.plugins.upload(path.join(blueprints_dir, plugin_wagon), '-y',
+                       path.join(blueprints_dir, plugin_yaml))
+    cfy.deployments.create(plugin_name, '-b', plugin_name)
+    _wait_for_deployment_creation(cfy, plugin_name)
+    execution_start_time = time.time()
+    mgr1.client.executions.start(plugin_name, 'install')
+
+    # check which management worker handles the execution
+    exec_node_id = json.loads(
```

Comment:
at this point i'd just use the rest client instead of the cli, to avoid the `strip`s (there's `mgr1.client`)